tox.ini

```diff
@@ -34,6 +34,15 @@ deps =
     pytest
     pytest-cov
 
+conda_deps =
+    windows: m2-base
+    windows: m2-make
+    windows: m2w64-toolchain
+    windows: libpython
+
+conda_channels =
+    windows: msys2
```

Comment:
The msys2 channel has been a default on Windows for some time: https://github.com/conda/conda/pull/2999